### PR TITLE
Replace export SQL and rename exported dialogs with modal

### PR DESCRIPTION
**Pull request by shucontech**
Signed-off-by: Saksham Gupta <shucontech@gmail.com>

Replace export SQL and rename exported dialogs with modal

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: templates/export.twig [hunk 1]
```diff
@@ -154,125 +168,140 @@
...
+              </td>
+              <td>
+                <button id="column_alias_button" class="btn btn-secondary" disabled="1">{% trans 'Add' %}</button>
+              </td>
+            </tr>
+          </table>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-secondary" id="saveAndCloseBtn" data-bs-dismiss="modal">
+            {% trans 'Save & close' %}
```
Comment (williamdes) [inline templates/export.twig:300]: not sure if this is right
```suggestion
            {% trans 'Save and close' %}
```

Comment (shucontech) [reply templates/export.twig:300]: The old version had `&` , so I just reproduced it.

Comment (williamdes) [inline templates/export.twig:300]: maybe it should be like all other modals, a save and a close button ?

Comment (shucontech) [reply templates/export.twig:300]: I'm not sure about that, the dialog had a `Save & Close` button. Should we split it into two buttons?

Comment (williamdes) [reply templates/export.twig:300]: Sorry about this, it was not very obvious but indeed it was `Messages.strSaveAndClose`

**Review by williamdes — APPROVED** [review]
LGTM

**Pull request merged**